scripts/DDL.sql

```diff
@@ -1,48 +1,75 @@
+CREATE SCHEMA jobs;
 
-CREATE TABLE users(
+CREATE TABLE jobs.users(
 	upn text PRIMARY KEY,
 	display_name text
 );
 
-CREATE TABLE tags(
+CREATE TABLE jobs.tags(
 	id serial PRIMARY KEY,
 	name text
 );
 
-CREATE TABLE roles(
+CREATE TABLE jobs.roles(
 	id serial PRIMARY KEY,
 	name text
 );
 
-CREATE TABLE standards(
+CREATE TABLE jobs.standards(
 	id serial PRIMARY KEY,
 	name text
 );
 
-CREATE TABLE advertisements(
+CREATE TABLE jobs.units(
 	id serial PRIMARY KEY,
-	role_id int REFERENCES roles(id) ON DELETE CASCADE,
-	unit text,
-	branch text,
-	department text,
+	name text UNIQUE
+);
+
+CREATE TABLE jobs.branches(
+	id serial PRIMARY KEY,
+	unit_id int REFERENCES jobs.units(id) ON DELETE CASCADE,
+	name text UNIQUE,
+	UNIQUE (id, unit_id)
+);
+
+CREATE TABLE jobs.departments(
+	id serial PRIMARY KEY,
+	branch_id int REFERENCES jobs.branches(id) ON DELETE CASCADE,
+	name text UNIQUE,
+	UNIQUE (id, branch_id)
+);	
+
+CREATE TABLE jobs.advertisements(
+	id serial PRIMARY KEY,
+	role_id int,
+	unit_id int,
+	branch_id int,
+	department_id int REFERENCES jobs.departments(id) ON DELETE CASCADE,
 	description text,
-	standard_id int REFERENCES standards(id) ON DELETE CASCADE,
 	entry_date Date,
 	seniority int,
 	is_damach boolean,
-	advertiser_upn text REFERENCES users(upn) ON DELETE CASCADE,
+	advertiser_upn text REFERENCES jobs.users(upn) ON DELETE CASCADE,
 	contact text,
```

Comment:
Change to:
`contact_upn text REFERENCES jobs.users(upn) ON DELETE CASCADE`

We want to store both the `name` of the person, and his `upn`

Reply:
I dont see way ti change it?

Reply:
Added in `trello`